lib/taggable.rb

```diff
@@ -267,7 +267,7 @@ def parse_tags(klass, incoming_tags)
 
   # Simple name to make it easier for people to use in full-text search
   def tag
-    (tags + filters).uniq.map{ |t| t.name }
+    (tags + filters).uniq.map{ |t| t.name == "Warning" ? "ArchiveWarning" : t.name}
```

Comment:
I'm not sure what the purpose of this change is. In production, the tag named "warning" is a non-canonical freeform whose name is all lowercase, so I'm not sure the check `t.name == "Warning"` will ever return true.

Reply:
Agreed. I'm not sure either. This was done on a previous branch (https://github.com/littlelines/otwarchive/commit/288a7694cddfd0247c9190b16f43cd7f1ec25600#diff-2d1fa506fd660847b60358f904056e8a) I will revert the change.